src/functionOverloading/CalculateIncome.java

```diff
@@ -0,0 +1,24 @@
+package functionOverloading;
+
+public class CalculateIncome {
+    public static void main(String[] args) {
+        System.out.println(calculateIncomeTax(2000));
+        System.out.println(calculateIncomeTax(2000, 0.30));
```

Comment:
i believe we should pass rate as percentage 20,30...
try to use one function in another